specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2023-12-10-preview/cloudhsm.json

```diff
@@ -0,0 +1,975 @@
+          "$ref": "#/definitions/RestoreProperties"
+        },
+        "backupProperties": {
+          "$ref": "#/definitions/BackupProperties"
+        }
+      },
+      "description": "Properties of a Cloud HSM Cluster.",
+      "type": "object"
+    },
+    "CloudHsmClusterPatchParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "The Cloud HSM Cluster's tags",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The Cloud HSM Cluster's tag's additional properties"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/CloudHsmClusterSku",
+          "description": "SKU details"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "Properties of the Cloud HSM Cluster",
+          "properties": {
+            "backupProperties": {
+              "$ref": "#/definitions/BackupProperties"
+            }
+          }
+        }
+      },
+      "description": "Patchable properties of the Cloud HSM Cluster",
+      "type": "object"
+    },
+    "CloudHsmClusterSecurityDomainProperties": {
+      "type": "object",
+      "description": "Security domain properties information for Cloud HSM cluster",
+      "properties": {
+        "fipsState": {
+          "format": "int32",
+          "type": "integer",
+          "description": "FIPS state information for security domain"
+        },
+        "activationStatus": {
+          "type": "string",
+          "description": "status of security domain activation"
+        }
+      }
+    },
+    "CloudHsmClusterSku": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster SKU information",
+      "required": [
+        "name",
+        "family"
+      ],
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "Sku family of the Cloud HSM Cluster",
+          "enum": [
+            "B"
+          ],
+          "x-ms-client-default": "B",
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuFamily",
+            "modelAsString": true
+          }
+        },
+        "name": {
+          "type": "string",
+          "description": "Sku name of the Cloud HSM Cluster",
+          "enum": [
+            "Standard_B1",
+            "Standard B10"
+          ],
+          "x-ms-enum": {
+            "name": "CloudHsmClusterSkuName",
+            "modelAsString": false
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Sku capacity"
+        }
+      }
+    },
+    "BackupProperties": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster backup information",
+      "properties": {
```

Comment:
"lastBackupDateTime" also needs to be part of PUT request model (even if it will not be processed by the API implementation)